packages/itwin/grouping-mapping-widget/src/components/Mappings/MappingsView.tsx

```diff
@@ -41,14 +41,15 @@ export const mappingViewDefaultDisplayStrings = {
 };
 
 export interface MappingsViewProps {
-  mappings: Mapping[];
+  mappings: Mapping[] | undefined;
```

Comment:
Does the requirement to support `undefined` come from the `react-query`? I don't think that it makes sense to render the `MappingsView` if you do not have the `Mappings`. The `mappings` property should remain mandatory.

Reply:
Mappings isn't immediately available, so I have to render something in its place while it loads. There can be a point where the mappings fetch can error out as well and what is returned is undefined.

Reply:
The component which requires the mappings defines what it needs. This component needs a mapping list, therefore if the mappings fetch errors out and returns undefined, it should be translated to an empty array there instead of this component handling undefined mappings list.

MappingsView should always receive a list to display. It can be empty, but it must be a list.

Reply:
Sure, fixed. I was initially steering towards that but this convinced me again.

Reply:
"Fixed", but I don't see any changes.

Reply:
git push connection refused 👎